# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/PATH_MAX.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-3.19/scripts/basic/fixdep.c.vanilla	2018-12-17 19:30:00.332000000 +0100
+++ linux-3.19/scripts/basic/fixdep.c	2018-12-17 19:32:33.084000000 +0100
@@ -106,6 +106,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
+#include <sys/param.h>
 #include <unistd.h>
 #include <fcntl.h>
 #include <string.h>
--- linux-3.19/scripts/kconfig/confdata.c.vanilla	2018-12-17 19:32:47.876000000 +0100
+++ linux-3.19/scripts/kconfig/confdata.c	2018-12-17 19:33:02.560000000 +0100
@@ -4,6 +4,7 @@
  */
 
 #include <sys/stat.h>
+#include <sys/param.h>
 #include <ctype.h>
 #include <errno.h>
 #include <fcntl.h>
--- linux-3.19/scripts/kconfig/zconf.tab.c_shipped.vanilla	2018-12-17 19:33:19.560000000 +0100
+++ linux-3.19/scripts/kconfig/zconf.tab.c_shipped	2018-12-17 19:33:54.508000000 +0100
@@ -85,6 +85,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <stdbool.h>
+#include <sys/param.h>
 
 #include "lkc.h"
 
